### ISSUE TITLE
app is not running properly

### ISSUE THREAD

**Issue by antonstavrou** [KhamisiKibet/24-Modern-Desktop-GUI#1]
i get this warning and the app no longer changes themes nor loads them properly. for example i have black and green colours which i did not add i can no longer locate the settings page either and thus cannot try to change theme either. I went with the instructions in the python video, I am at around 1:20:00

python main.py
Live monitoring Qss/scss/defaultStyle.scss file for changes
Live monitoring json-styles/style.json for changes
C:\Users\something\AppData\Roaming\Python\Python312\site-packages\Custom_Widgets\JSonStyles\__init__.py:842: RuntimeWarning: Failed to disconnect (None) from signal "clicked()".
  button.clicked.disconnect()
all icons have been checked and missing icons generated!

**Comment by CharaD7** [comment]
Hello, @antonstavrou

Can you confirm if you see the src folder in your root directory?

**Comment by antonstavrou** [comment]
Yes I do. Issues started somewhere between inserting the different colour themes (dark yellow never worked for me) and the promotion of the three menus. Hope this helps.

Edit: I have gone back a few steps and reintroduced each element of the code back and it's working fine. the error message above still exists but otherwise working right. I have no idea what exactly fixed it as I literally cut and pasted the code I had written. Basically "switched it off and back on again". If anyone has any ideas about the error message though I would appreciate any ideas.
